storage/2020-08-04/datalakestore/paths/options.go

```diff
@@ -0,0 +1,21 @@
+package paths
+
+import (
+	"github.com/hashicorp/go-azure-sdk/sdk/client"
+	"github.com/hashicorp/go-azure-sdk/sdk/odata"
+)
+
+type pathOptions struct {
+}
+
+func (p pathOptions) ToHeaders() *client.Headers {
+	return nil
+}
+
+func (p pathOptions) ToOData() *odata.Query {
+	return nil
+}
+
+func (p pathOptions) ToQuery() *client.QueryParams {
+	return nil
+}
```

Comment:
rather than returning `nil` for everything, we can instead specify `nil` rather than `pathOptions{}` for the `Options` object, fwiw